device/iot-device-samples/pnp-device-sample/temperature-controller-device-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/device/TemperatureController.java

```diff
@@ -178,23 +178,19 @@ public static void main(String[] args) throws IOException, URISyntaxException, P
         maxTemperature.put(THERMOSTAT_1, 0.0d);
         maxTemperature.put(THERMOSTAT_2, 0.0d);
 
-        new Thread(new Runnable() {
-            @SneakyThrows({InterruptedException.class, IOException.class})
```

Comment:
q - were these "throws" declarations redundant, given that our `main` already declares that it throws these two exceptions?

Reply:
Actually these are required. The build breaks if these are left. Will need to suppress.